Capstone/Assets/Game/Scripts/Enemy_SideToSide.cs

```diff
@@ -15,26 +15,45 @@ public class Enemy_SideToSide : Character
         private bool damaged = false;
         private float timer = 0f;
         private float attackTimer = 0f;
-        private float moveTimer = 0f;
-        private int flip = 1;
+        private int index;
 
-      
-        private void Update()
+        [SerializeField][Range(0, 1)]
+        private float speed;
+        public List<Vector3> movePoints = new List<Vector3>();
+
+        // Incase we want to use transforms instead of Vector3's
+        // We will need to do some code changes below but it should be similar
+        //public List<Transform> movePoints = new List<Transform>();
+
+        public void Start()
         {
+            transform.position = movePoints[0];
+            index = 0;
+            player = GameObject.Find("Player").GetComponent<Player>();
```

Comment:
This should probably just be set in the Inspector rather than this. Or we should get it from another object or something that creates these.